src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+type Props = {
+  todos: Todo | null;
```

Comment:
The prop name `todos` should be singular, such as `todo`, since it represents a single todo item. Update the prop name to better reflect its purpose.